### PR TITLE
Fixes Install/Readme

### PR DESCRIPTION
**Pull request by anselmbradford**
## Changes

- Fixed typos, etc., for `INSTALL.md` missed in https://github.com/cfpb/cfgov-refresh/pull/1534
- Brought back stand-alone install section as there's content that needs to be removed from `README.md` too, but I'm not sure we're off stand-alone installations team-wide yet.

## Review

- @jimmynotjim 
- @sebworks 
- @KimberlyMunoz 

## Todos

- README.md and INSTALL.md should be updated to only be the Vagrant setup after a fresh run-through of those steps in INSTALL.md.

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -1,14 +1,12 @@
-# Installation and Congiguration for cfgov-refersh with Vagrant
```
Comment (KimberlyMunoz) [inline INSTALL.md]: How has noone noticed this before?

Comment (anselmbradford) [inline INSTALL.md]: Yeah https://github.com/cfpb/cfgov-refresh/pull/1534#discussion-diff-56154272

**Comment by kave** [comment]
vary noice

**Comment by sebworks** [comment]
:+1:

**Pull request merged**